### PR TITLE
Jenkins cleanup and OVA bootstrap fixes.

### PR DESCRIPTION
**Pull request by sradhanand-pati**

### CODE REVIEW

File: devops/ci/multi_node_deploy/3node_destroy.jenkinsfile [hunk 1]
```diff
@@ -122,26 +122,31 @@ node {
...
-                    yum clean all
-                    rm -rf /var/cache/yum
-                    # Remove cortx-py-utils
-                    pip3 uninstall -y cortx-py-utils
                     # Cleanup pip packages
                     pip3 freeze|xargs pip3 uninstall -y
-                    # Cleanup pip config
-                    test -e /etc/pip.conf && rm -f /etc/pip.conf
-                    test -e ~/.cache/pip && rm -rf ~/.cache/pip || true
+                    yum remove -y python3
```
Comment (ypise) [inline devops/ci/multi_node_deploy/3node_destroy.jenkinsfile:136]: Remove this line. It's not required and risky.

File: devops/ci/single_node_deploy/destroy.jenkinsfile [hunk 2]
```diff
@@ -104,29 +104,30 @@ node {
...
+                    pip3 uninstall -y cortx-py-utils
                     yum erase -y glusterfs-fuse glusterfs-server glusterfs     # Gluster FS packages
                     yum erase -y salt-minion salt-master salt-api   # Salt packages
                     yum erase -y python36-m2crypto                  # Salt dependency
                     yum erase -y python36-cortx-prvsnr              # Cortx Provisioner API packages
                     # Brute clean for any cortx rpm packages
                     yum erase -y *cortx*
+                    # Cleanup pip packages
+                    pip3 freeze|xargs pip3 uninstall -y
+                    yum remove -y python3
```
Comment (ypise) [inline devops/ci/single_node_deploy/destroy.jenkinsfile:117]: Same here, don't remove python3

File: devops/ci/single_node_deploy/destroy.jenkinsfile [hunk 3]
```diff
@@ -104,29 +104,30 @@ node {
...
                 """
                 echo "Successfully uninstalled rpms."
             }
 
             stage("Cleanup bricks and other directories") {
                 sshCommand remote: remote, command: """
+                    test -e /var/cache/yum && rm -rf /var/cache/yum
+                    # Cleanup pip config
+                    test -e /etc/pip.conf && rm -f /etc/pip.conf
+                    rm -rf ~/.cache/pip
```
Comment (ypise) [inline devops/ci/single_node_deploy/destroy.jenkinsfile:130]: It's safe to add a check:
```
test -e ~/.cache/pip && rm -rf ~/.cache/pip
```

File: devops/ci/multi_node_deploy/3node_destroy.jenkinsfile [hunk 4]
```diff
@@ -122,26 +122,31 @@ node {
...
                 """
                 echo "Successfully uninstalled rpms."
             }
 
             stage("${remote.name}: Cleanup bricks and other directories") {
                 sshCommand remote: remote, command: """
+                    test -e /var/cache/yum && rm -rf /var/cache/yum
+                    # Cleanup pip config
+                    test -e /etc/pip.conf && rm -f /etc/pip.conf
+                    rm -rf ~/.cache/pip
```
Comment (ypise) [inline devops/ci/multi_node_deploy/3node_destroy.jenkinsfile:149]: It's safe to add a check:
```
test -e ~/.cache/pip && rm -rf ~/.cache/pip
```

**Review by ypise — CHANGES_REQUESTED** [review]

File: devops/ci/multi_node_deploy/3node_destroy.jenkinsfile (hunk 1 above)
Comment (sradhanand-pati) [reply devops/ci/multi_node_deploy/3node_destroy.jenkinsfile]: Ok

**Review by ypise — APPROVED** [review]

**Review by ypise — APPROVED** [review]

**Pull request merged**